File: KotlinMobileApp/Java/Start/app/src/main/java/com/rachelleboyette/catquiz/interfaces/CatFactApi.java
package com.rachelleboyette.catquiz.interfaces;

import com.rachelleboyette.catquiz.dtos.CatFactDto;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface CatFactApi {

    @GET("/fact?")
    Call<CatFactDto> getCatFact(@Query("max_length")int maxLength);
}

File: KotlinMobileApp/KotlinMobileApp/app/src/main/java/com/neudesic/catquiz/interfaces/CatFactApi.kt
package com.neudesic.catquiz.interfaces

import com.neudesic.catquiz.dtos.CatFactDto
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface CatFactApi {
    @GET("/fact?")
    fun getCatFact(@Query("max_length") maxLength: Int): Call<CatFactDto>
}